Lib/fontmake/font_project.py

```diff
@@ -450,10 +450,10 @@ def subset_otf_from_ufo(self, otf_path, ufo):
 
         for old_name, new_name in zip(ufo_order, ot_order):
             glyph = ufo[old_name]
-            if ((keep_glyphs and old_name not in keep_glyphs) or
-                not glyph.lib.get(GLYPHS_PREFIX + 'Glyphs.Export', True)):
-                continue
-            include.append(new_name)
+            in_keep_glyphs = keep_glyphs and old_name not in keep_glyphs
+            exported = glyph.lib.get(GLYPHS_PREFIX + "Glyphs.Export", True)
+            if in_keep_glyphs or exported:
+                include.append(new_name)
```

Comment:
oh, since you're touching this, I think you should have a look at this conversation with @schriftgestalt
https://github.com/googlei18n/fontmake/issues/381
and 
https://github.com/googlei18n/glyphsLib/issues/295

I forgot the details about it and don't have time to investigate now